addons/loyalty/wizard/loyalty_card_update_balance.py

```diff
@@ -30,10 +30,20 @@ def action_update_card_point(self):
         else:
```

Comment:
```suggestion
        else:
```
One of the ideas, maybe the above "if" should also take into account if old balance was negative. And if it was negative, then it should also assign points to "used" value instead of issued.